prow/cmd/build/main.go

```diff
@@ -158,12 +231,22 @@ func main() {
 	o := parseOptions()
 	logrusutil.NewDefaultFieldsFormatter(nil, logrus.Fields{"component": "build"})
 
-	configs, defaultContext, err := contextConfigs(o.kubeconfig)
+	pjNamespace := ""
+	if o.config != "" {
+		pc, err := config.Load(o.config, "") // ignore jobConfig
```

Comment:
Why not the agent?

Reply:
I worry downstream users will use other parts of the config later, other than the namespace, and not realize it's not reloading.

Reply:
How? The scope of the config object is this if statement.